src/pages/docs/chat/rooms/message-reactions.mdx

```diff
@@ -120,14 +292,34 @@ Always call `Message.with(event)` when applying message events and reaction even
 
 ## Subscribing to message reactions <a id="subscribe"/>
 
+<If lang="javascript,swift,kotlin">
 Ably generates a summary (aggregate) of the reactions for each message and for each reaction type. For displaying accurate counts for message reactions, subscribe to changes in the message summary.
+</If>
+
+<If lang="react">
+Subscribe to message reactions with the [`useMessages`](https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/functions/chat-react.useMessages.html) hook. Supply a `reactionsListener` to receive message reaction summary events. Ably generates a summary (aggregate) of the reactions for each message and for each reaction type. For displaying accurate counts for message reactions, subscribe to changes in the message summary.
+</If>
 
 <Code>
 ```javascript
 room.messages.reactions.subscribe((event) => {
     console.log("received reactions summary event", event);
 });
 ```
+
+```react
+import { useMessages } from '@ably/chat/react';
+
+const MyComponent = () => {
+  useMessages({
+    reactionsListener: (event) => {
+      console.log("received reactions summary event", event);
+    },
+  });
+
+  return <div>...</div>;
+};
+```
```

Comment:
Not this PR, but the `@` in the timeserial comes out a bit odd in the table below